src/test/java/org/broadinstitute/hellbender/BwaMemIntegrationTest.java

```diff
@@ -2,22 +2,20 @@
 
 import org.broadinstitute.hellbender.utils.bwa.BwaMemIndex;
 import org.broadinstitute.hellbender.utils.test.BaseTest;
-import org.testng.annotations.AfterSuite;
-import org.testng.annotations.BeforeSuite;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
 
 public class BwaMemIntegrationTest extends BaseTest {
 
     private BwaMemIndex index;
 
-    @BeforeSuite
+    @BeforeClass
     public void loadIndex() {
```

Comment:
Does this really take minutes?  that's gross... we should try to change it uses a smaller test file if that wouldn't reduce the power of the test unreasonably.

Reply:
It seems to take about 2 minutes on my laptop.

Reply:
I am suspecting this is leading to the new indexing step I see when running tests locally, and does increase the test time by ~3min.